### ISSUE TITLE
get the name of focus cell

### ISSUE THREAD

**Issue by ThanhLNP** [ONLYOFFICE/document-server-integration#127]
I have an idea that when you select a cell, you will get the name and value of that cell
and I can fill in a certain value in the cell I am selecting
example: When I click on cell B2 I get it address "B2", and it value is "hello"
![image](https://user-images.githubusercontent.com/52529251/109451965-414f0680-7a81-11eb-841c-675d546c396c.png)

I am using sample python code so how can I write it in file editor.html

Thank you for reading this question

**Comment by ShockwaveNN** [comment]
I think you should use our marcos system and take a look at `GetValue` method, which described [here](https://api.onlyoffice.com/docbuilder/spreadsheetapi/apirange/getvalue)

**Comment by ThanhLNP** [comment]
First of all, thank you
Is there any way to automatically use macros on mouse clicks?
looks like this: https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_event_mouse_button

**Comment by ShockwaveNN** [comment]
@ThanhLNP I don't think so, as I am aware our macroses only support changing content of document

**Comment by ThanhLNP** [comment]
So is there no way for me to write this 
Api.GetSelection().GetAddress(false, false, 'xlA1', false)
in file editor.html?

**Comment by ShockwaveNN** [comment]
That in the main purpose of your script? You tell what you want to do, but I still not understand why you want to do those steps? Maybe there is simpler solution for your taks without any scripts

**Comment by ThanhLNP** [comment]
I have an idea to do "text to speech" every time I click on a certain cell
So maybe I will create a plugin but it cannot be automatically speech
